content/_en/landing/create_an_account.md

```diff
@@ -3,37 +3,36 @@ layout: landing
 title: Create an account
 description: Join the millions of people who trust Login.gov for safe, secure
   access to government agencies.
-one_account_banner: true
+steps:
+  class: create-an-account
+  intro: "When you’re ready to create your secure login.gov account, you’ll need
+    to provide a few pieces of information:"
+  step1: >-
+    ## 1. Email address
+      * We recommend a personal email address that you’ll always be able to access rather than a work email address.
+      * If you already have an account with login.gov with that email address, we’ll send you an email to let you know how you can reset your password and access the account.
```

Comment:
should we unindent these? looks like spanish is unindented

```suggestion
    * We recommend a personal email address that you’ll always be able to access rather than a work email address.
    * If you already have an account with login.gov with that email address, we’ll send you an email to let you know how you can reset your password and access the account.
```